src/interface/src/app/map/map.component.spec.ts

```diff
@@ -8,7 +8,9 @@ import { MatCheckboxModule } from '@angular/material/checkbox';
 import { MatCheckboxHarness } from '@angular/material/checkbox/testing';
 import { MatDialog, MatDialogRef } from '@angular/material/dialog';
 import { MatRadioModule } from '@angular/material/radio';
-import { MatRadioGroupHarness } from '@angular/material/radio/testing';
+import {
```

Comment:
strange formatting here - remove the comma and reformat?

Reply:
Nice catch!